### ISSUE TITLE
[INSTALL] (Pop!_OS) FiftyOneConfigError: MongoDB could not be installed on your system. Please define a `database_uri` in your `fiftyone.core.config.FiftyOneConfig` to connect to yourown MongoDB instance or cluster 

### ISSUE THREAD

**Issue by p-a-a-a-trick** [voxel51/fiftyone#4605]
I am having an error when attempting to follow along with [a tutorial](https://docs.voxel51.com/tutorials/evaluate_classifications.html) using my own image directory.

Similar issue here, probably similar fix? #4427 

This is not urgent; I am considering a different OS anyways. I think there are some enterprises that use Pop!_OS, though, so this might still be worth fixing.

Error:

```python
FiftyOneConfigError: MongoDB could not be installed on your system. Please define a `database_uri` in your `fiftyone.core.config.FiftyOneConfig` to connect to yourown MongoDB instance or cluster
```

I have tried creating a new env and upgrading pip. Please let me know if I'm reporting this in error or if I should try something else.

Thanks!

### System information

-   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04): Linux Pop!_OS 22.04 LTS
-   **Python version** (`python --version`): 3.10.12
-   **FiftyOne version** (`fiftyone --version`):  0.24.1
-   **FiftyOne installed from** (pip or source): pip

```python
>>> import csv
>>> import os
>>> import pathlib
>>> import platform
>>> path = pathlib.Path("/etc/os-release")
>>> with open(path) as stream:
...         reader = csv.reader(stream, delimiter="=")
...         # filter empty lines
...         d = dict(line for line in reader if line)
... 
>>> print(d)
{'NAME': 'Pop!_OS', 'VERSION': '22.04 LTS', 'ID': 'pop', 'ID_LIKE': 'ubuntu debian', 'PRETTY_NAME': 'Pop!_OS 22.04 LTS', 'VERSION_ID': '22.04', 'HOME_URL': 'https://pop.system76.com', 'SUPPORT_URL': 'https://support.system76.com', 'BUG_REPORT_URL': 'https://github.com/pop-os/pop/issues', 'PRIVACY_POLICY_URL': 'https://system76.com/privacy', 'VERSION_CODENAME': 'jammy', 'UBUNTU_CODENAME': 'jammy', 'LOGO': 'distributor-logo-pop-os'}
```

### Commands to reproduce

As thoroughly as possible, please provide the Python and/or shell commands used to encounter the issue.

```bash
patrick@pop-os:~/python$ python3 -m venv create fiftyone
patrick@pop-os:~/python$ source fiftyone/bin/activate
(fiftyone) patrick@pop-os:~/python$ pip install --upgrade pip setuptools wheel build
(fiftyone) patrick@pop-os:~/python$ pip install fiftyone
(fiftyone) patrick@pop-os:~/python$ pip install torch torchvision ipython
(fiftyone) patrick@pop-os:~/python$ ipython
```

```python
In [1]: import os
   ...: import fiftyone as fo
   ...: # dir of jpgs
   ...: DATASET_DIR = os.path.join(os.getcwd(), "img_dir/")
   ...: 
   ...: dataset = fo.Dataset.from_dir(
   ...:     DATASET_DIR,
   ...:     fo.types.ImageClassificationDirectoryTree,
   ...:     name="imgs"
   ...: )

---------------------------------------------------------------------------
ServiceExecutableNotFound                 Traceback (most recent call last)

# <snip>, let me know if you need this

FiftyOneConfigError: MongoDB could not be installed on your system. Please define a `database_uri` in your `fiftyone.core.config.FiftyOneConfig` to connect to yourown MongoDB instance or cluster
```

### Describe the problem

Created a new environment, upgraded pip+installed wheel, installed fiftyone, torch, and torchvision through pip. Opened ipython, ran the above block from the tutorial linked previously.

### Other info/logs
N/A

**Comment by benjaminpkane** [comment]
Hi @p-a-a-a-trick. We can publish a new `fiftyone-db` with pop os support. In the meantime, you can also configure your own [database_uri](https://docs.voxel51.com/user_guide/config.html#configuring-mongodb-connection)

**Comment by p-a-a-a-trick** [comment]
@benjaminpkane that worked great. Thank you!

For anyone (like me) that hasn't used mongodb before / is hesitant to self-manage it and needs a workaround before this gets built:

1. [install mongodb](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-ubuntu/#std-label-install-mdb-community-ubuntu)
2. start mongodb following the start directions
3. find the host/port. I did this on accident by running `mongosh`. ("`Connecting to:      mongodb://<host>:<port>/blahblahblah`")
4. create a venv, upgrade pip, install fiftyone torch torchvision etc
5. `export FIFTYONE_DATABASE_URI=mongodb://<host>:<port>`
6. open python runner of your choice
7. do what you gotta do

Small edit to what I did to load my dataset, though - I just used `dataset = fo.Dataset.from_images_patt` instead.

Thanks again!
